### ISSUE TITLE
Accessibility issues main navigation (category type folder and flyout not accessible with tab navigation)

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#4063]
When a main navigation item is not also a category link but just a structural element, it cannot be accessed via tab navigation. Therefore the underlying flyout menu cannot be opened.

I suggest to add tabindex=0 to the main-navigation-link div when category.type is folder. And also to add focus event as opening trigger in flyout menu plugin.

**Comment by shopwareBot** [comment]
[public] Automated response: This issue is linked to the internal issue [https://shopware.atlassian.net/browse/NEXT-32669](https://shopware.atlassian.net/browse/NEXT-32669).

 

[created from NEXT-32669, comment 485404]

**Comment by shopwareBot** [comment]
[public] Hello,

 Within our company roadmap and work capacity, we try to address each bug or improvement request but admit that not each one will be resolved. To continue our culture of honesty and openness, we are closing this issue to focus on our roadmap on behalf of all Shopware users.

 If you feel like this issue is still impacting you, please create a new issue and let us know. Thank you again for your valuable feedback.



[created from NEXT-32669, comment 492310]
